### ISSUE TITLE
Unable to apply effects to voice recordings

### ISSUE THREAD

**Issue by 5trongthany** [wireapp/wire-android#3338]
**Describe the bug**
When trying to record and send audio, applying a filter to voice causes app to crash.

**To Reproduce**
record audio in any conversation, once completed tap any of the voice effects options.

1. Go to any conversation
2. Tap option to record voice message
3. Record voice message
4. Tap any filter option

**Expected behavior**
Voice filter would be applied, then played back for user to review

**Screenshots**

**Smartphone (please complete the following information):**
- Wire version that exhibits the issue: 3.68.991 Android
- Android OS version used to run Wire: Android 10
- Is this a custom firmware or a stock one: Stock
- Mobile phone model/manufacturer: Fairphone 3
- Mobile network type (EDGE/LTE/Wi-Fi/Offline): Network agnostic issue

**Comment by DrAxelbauer** [comment]
Yeah I am also encountering this issue since years now...It seems a little bit, that the development and debugging has become slow these days...

**Comment by 5trongthany** [comment]
@DrAxelbauer primary reason I've switched to Mattermost, no new development from wire.
